package.json

```diff
@@ -5,7 +5,7 @@
     "build:all:dev": "npm run build:app:dev && npm run build:server",
     "build:all:prod": "npm run build:app:prod && npm run build:server",
     "build:app:dev": "ng build --configuration development",
-    "build:app:prod": "ng build --localize --configuration production",
+    "build:app:prod": "ng build --localize --configuration production && cp src/redirect.html dist/harassment-manager/index.html",
```

Comment:
Why do we have to manually copy `redirect.html`?

Reply:
When the app is built, it will create the files in `en` and `fr` directories, however when we deploy, App Engine will look for `index.html` in the base directory. We use the `redirect.html` to redirect it to `en` folder. We've tried using [`app.yml`](https://github.com/CodeForAfrica/harassment-manager/blob/875a627db8032c7622133561db252b212779bb4e/app.yaml) to automatically redirect but we couldn't get it to work.